libs/@guardian/source-foundations/src/utils/typography.ts

```diff
@@ -0,0 +1,16 @@
+import type { TypographyPreset } from '../typography/types';
+
+/*
+ * Convert `font-size` value in typography preset from rem to px
+ */
+export const presetToPx = (preset: TypographyPreset) => {
+	const REGEX_FONT_SIZE = /font-size:\s(\d+\.\d+)rem/;
```

Comment:
I think you might have an issue if the value does not contain a decimal part…

```suggestion
	const REGEX_FONT_SIZE = /font-size:\s(\d+\.?\d*)rem/;
```